container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/package_private_helpers.go

```diff
@@ -0,0 +1,105 @@
+package logs_aggregator_functions
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/docker_manager"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/docker_manager/types"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/object_attributes_provider/label_key_consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/object_attributes_provider/label_value_consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/container_status"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/logs_aggregator"
+	"github.com/kurtosis-tech/stacktrace"
+	"net"
+)
+
+const (
+	shouldShowStoppedLogsAggregatorContainers = true
+)
+
+func getLogsAggregatorObjectAndContainerId(
+	ctx context.Context,
+	dockerManager *docker_manager.DockerManager,
+) (
+	resultMaybeLogsAggregator *logs_aggregator.LogsAggregator,
+	resultMaybeContainerId string,
+	resultErr error,
+) {
+	allLogsAggregatorContainers, err := getAllLogsAggregatorContainers(ctx, dockerManager)
+	if err != nil {
+		return nil, "", stacktrace.Propagate(err, "An error occurred getting all logs Aggregator containers")
+	}
+
+	if len(allLogsAggregatorContainers) == 0 {
+		return nil, "", nil
+	}
+	if len(allLogsAggregatorContainers) > 1 {
```

Comment:
Ah yes exactly, so I'd just push this check and error in `getAllLogsAggregatorContainers` directly

Reply:
done